data/projects.json

```diff
@@ -148,6 +148,16 @@
         "level": "basic",
         "site": "",
         "tags": "python-project,personal-assistant,python-assistant"
+    },
+    {
+        "name": "Gorcery-FinderApp",
```

Comment:
Please check this repo is exists or is with public visibility

Reply:
@Dustin1386 , Please mention your exact repository name for the "name" key as it will be used as a link to your repository, mentioning another name will cause a 404 error.

Reply:
@Dustin1386 , Please mention your exact repository name for the "name" key as it will be used as a link to your repository, mentioning another name will cause a 404 error.